cmd/agent/dist/conf.d/jmx.d/conf.yaml.example

```diff
@@ -29,6 +29,13 @@ instances:
   #   key_store_password: password
   #   rmi_registry_ssl: false # Optional, should be set to true if "com.sun.management.jmxremote.registry.ssl" is set to true on the target JVM.
   #
+  #   # Use better metric names for garbage collection metrics. The default value is false to ensure backward compatibility.
```

Comment:
just to be extra-clear, I'd write `Set to true to use better metric names [...]`